assets/javascripts/discourse/templates/admin/plugins-slack.hbs

```diff
@@ -2,16 +2,22 @@
   <h1>{{i18n "slack.header.rules"}}</h1>
   <table>
     <tr>
-      <th>{{i18n "slack.category"}}</th>
       <th>{{i18n "slack.channel"}}</th>
+      <th>{{i18n "slack.category"}}</th>
+      {{#if taggingEnabled}}
+        <th>{{i18n "slack.tags"}}</th>
+      {{/if}}
```

Comment:
Note that the styling on the table header is off.

![screenshot from 2017-03-14 17-15-01](https://cloud.githubusercontent.com/assets/4335742/23893529/ca37dbe8-08d9-11e7-9955-96bffaa08d1d.png)